lib/src/signed_video_h26x_internal.h

```diff
@@ -29,6 +29,8 @@
 typedef struct _h26x_nalu_list_item_t h26x_nalu_list_item_t;
 typedef struct _h26x_nalu_t h26x_nalu_t;
 
+#define NR_OF_PENDING_GOPS 120
```

Comment:
MAX_PENDING_GOPS
otherwise when reading code it could be interpreted as how many pending GOPs we have right now.